lightning/src/chain/channelmonitor.rs

```diff
@@ -226,7 +226,7 @@ pub(crate) const LATENCY_GRACE_PERIOD_BLOCKS: u32 = 3;
 /// It may cause spurrious generation of bumped claim txn but that's allright given the outpoint is already
 /// solved by a previous claim tx. What we want to avoid is reorg evicting our claim tx and us not
 /// keeping bumping another claim tx to solve the outpoint.
-pub(crate) const ANTI_REORG_DELAY: u32 = 6;
+pub const ANTI_REORG_DELAY: u32 = 6;
```

Comment:
Maybe make the last four lines a comment instead of documentation? Feels weird to expose them.